### PR TITLE
Marketplace finish sidebar

### PR DESCRIPTION
**Pull request by hwang635**
- Implement filtering by item type (standard vs agency type)
- Ignore privatised records to hide from display
- Implement sorting w/ sort by menu

### CODE REVIEW

File: src/components/Marketplace/MarketplaceScreen.js [hunk 1]
```diff
@@ -78,33 +82,50 @@ export default function MarketplaceScreen() {
             produceType,
             palletsAvailable: record.fields['pallets available'] || 0,
             hasAgencyPrice,
+            // fields for sorting
+            'sell by date': record.fields['sell by date'] || 0,
+            'first available date': record.fields['first available date'],
```
Comment (curesh) [inline src/components/Marketplace/MarketplaceScreen.js]: Is there a reason why the 'sell by date' field is ORed with 0, but the other fields aren't?

File: src/components/Marketplace/MarketplaceScreen.js [hunk 2]
```diff
@@ -78,33 +82,50 @@ export default function MarketplaceScreen() {
...
   const farmSeasonFilters = ['Fall', 'Winter', 'Summer', 'Spring'];
   const [itemsPerFarmSeason, setSeasonItems] = useState([]);
 
+  // Manages item type filtering
+  const [checkedItemTypes, setCheckedItemTypes] = useState([]);
+  const onItemFilterChange = (newFilters) => {
+    setCheckedItemTypes(newFilters);
+  };
+  const itemTypeFilters = ['Agency Price', 'Standard Items'];
+  const [itemsPerItemType, setItemsPerItemType] = useState([]);
```
Comment (curesh) [inline src/components/Marketplace/MarketplaceScreen.js:116]: Maybe define all the state variables together at the top of this func? Just to make it consistent with how you've defined them in the function above. Up to you though

File: src/components/Marketplace/MarketplaceScreen.js [hunk 3]
```diff
@@ -365,7 +416,7 @@ export default function MarketplaceScreen() {
               />
             ))
           }
-          {/* Map each array of produceListing info to render a ProduceCard */
+          {/* Map each array of produceListing info to render a Pr oduceCard */
```
Comment (curesh) [inline src/components/Marketplace/MarketplaceScreen.js]: Typo in 'Pr oduceCard'

**Review by curesh — APPROVED** [review]
Wow this feature culminates the behemoth marketplace page that you've been working on, and seeing all these different components come together looks super cool! Your code is super clean as usual, but you already knew that– I've just left a few comments on some small syntactical stuff. Otherwise, amazing job

File: src/components/Marketplace/MarketplaceScreen.js (hunk 1 above)
Comment (hwang635) [reply src/components/Marketplace/MarketplaceScreen.js]: Nope, I forgot to add null checks for the rest. Fixed it!

**Pull request merged**